package.json

```diff
@@ -6,6 +6,7 @@
     "uglify-js": "2.4.24"
   },
   "devDependencies": {
-    "jshint": "^2.7.0"
+    "jshint": "^2.7.0",
+    "edx-custom-a11y-rules": "edx/edx-custom-a11y-rules"
```

Comment:
Oh! I had only used it with a hash. Didn't realize it was that easy for github. Cool.

An npm package is a good idea but not necessary for this round of work.